static/css/main.ie.css

```diff
@@ -17024,6 +17125,9 @@ a.focus .summary_event-date {
   display: inline-block;
 }
 
+/* Event-Footer
+   ========================================================================== */
+
 .event-footer {
   font-family: "AvenirNextLTW01-Regular", Arial, sans-serif;
```

Comment:
Can `.webfont-regular()` be used here? https://github.com/cfpb/cf-core/blob/b0dd19e020015595e84ec56418040fc84e478902/src/cf-base.less#L46